build.gradle

```diff
@@ -7,6 +7,13 @@
 
 import aQute.lib.io.IO
 
+apply plugin: 'com.gradle.build-scan'
+
+buildScan {
+    termsOfServiceUrl   = 'https://gradle.com/terms-of-service'
+    termsOfServiceAgree = 'yes'
```

Comment:
This also make a blanket assumption the everyone accepts the terms of service. A developer should really make this decision for them self.

Reply:
I've added a check that solves this problem. Scans are enabled by default when run on CI, but the terms of service are checked if that's not the case. Developers have the choice to accept or refuse the terms.